gnucash/gnome-utils/assistant-xml-encoding.c

```diff
@@ -1091,7 +1091,7 @@ gxi_parse_file (GncXmlImportData *data)
     }
 
     xaccLogDisable ();
-    gxi_update_progress_bar (_("Reading file..."), 0.0);
```

Comment:
No, this are status messages. Here the Ellipses means:
Work in progress, be patient!

Reply:
> No, this are status messages. Here the Ellipses means:
> Work in progress, be patient!
> [Edit:]
> Often further details or a progress bar follow.

Yeah , it's suitable here. I made a mistake.